packages/devtools_app/lib/src/standalone_ui/vs_code/devices.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import '../api/vs_code_api.dart';
+
+class Devices extends StatelessWidget {
+  const Devices(
+    this.api,
+    this.devices, {
+    required this.selectedDeviceId,
+    super.key,
+  });
+
+  final VsCodeApi api;
+  final List<VsCodeDevice> devices;
+  final String? selectedDeviceId;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
```

Comment:
we likely want to change this to use a table / list with selectable rows for reference in the next ui iteration

Reply:
We're already using a table (this column is to put the heading above it). I've added a TODO (further down in `_createDeviceRow`) about using a highlighted row instead of label.